packages/deployment/init.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-continue */
+/* eslint-disable no-await-in-loop */
```

Comment:
Placing these exceptions here makes me uncomfortable. We should only enforce these rules if we feel strongly enough that they're the right convention that we want to them to apply in all but exceptional circumstances. We should either require documenting why an exception is necessary in a particular case, or drop the rule.

I think no-continue is a fine rule, and I would favor enforcing it here. 

My impression is that no-await-in-loop is generally a good rule, and exceptions should be documented locally where they occur and not globally over a file.

Reply:
I can't remember the last time I used a `continue` so I haven't thought much about it. What's wrong with using `continue`?

Reply:
I don't have a specific problem with `continue`. I use it occasionally. The uses in this file were trivially replaced producing clearer code. If someone thinks that's always the case, I'm willing to try and willing to override with an explicit explanation when the rule is wrong.